libs/backend/core/model/src/lib/query.model.ts

```diff
@@ -0,0 +1,4 @@
+export class QueryPaginationModel {
+  offset?: number;
+  limit?: number;
+}
```

Comment:
par tout hasard tu peux pas faire ca pour simplifier le code ? 
```suggestion
export class QueryPaginationModel {
  offset?: number = PAGINATION_BASE_OFFSET;
  limit?: number = PAGINATION_BASE_LIMIT;
}
```

Reply:
A tester